config/settings/base.py

```diff
@@ -125,6 +125,56 @@
 SERVER_EMAIL = DEFAULT_FROM_EMAIL
 
 
+# Logging
```

Comment:
This is good!  I might have put this in a separate PR, but for our purposes this is fine, we can leave it here.